muvsfunc.py

```diff
@@ -5525,8 +5525,8 @@ def S_BoxFilter(clip: vs.VideoNode, radius: int = 1, planes: PlanesType = None)
     return res
 
 
-def VFRSplice(clips: Sequence[vs.VideoNode], tcfile: Optional[Union[str, os.PathLike]] = None,
-              v2: bool = True, precision: int = 6) -> vs.VideoNode:
+def VFRSplice(clips: Sequence[vs.VideoNode], ref: vs.VideoNode = None, tcfile: Optional[Union[str, os.PathLike]] = None,
```

Comment:
Please always add new parameters at the end of the parameter list for compatibility with existing scripts.